installations/src/android/installations_android.cc

```diff
@@ -186,6 +187,19 @@ void InstallationsInternal::Cleanup() {
   }
 }
 
+void InstallationsInternal::LogHeartbeat(const firebase::App& app) {
+  // Calling the native getter is sufficient to cause a Heartbeat to be logged.
+  JNIEnv* env = app.GetJNIEnv();
+  jobject platform_app = app.GetPlatformApp();
+  jclass installations_class = installations::GetClass();
```

Comment:
Is there a chance that installations_class could be null here? If so, it would be good to check and avoid calling GetInstance.

Reply:
What can cause a jclass to be null? I didn't expect there to be any challenge with getting a reference to a class in order to call a static method.

This logic is roughly the same as InstallationsInternal above, though that one also does some additional cleanup or initialization because it is the first method called.